packages/cozy-stack-client/src/__tests__/CozyStackClient.spec.js

```diff
@@ -247,6 +261,39 @@ describe('CozyStackClient', () => {
       }
     })
   })
+
+  describe('refreshToken', () => {
+    beforeAll(() => {
+      global.fetch = require('jest-fetch-mock')
+    })
+
+    beforeEach(() => {
+      fetch.resetMocks()
+      fetch.mockResponse(FAKE_APP_HTML)
+    })
+
```

Comment:
I think the response.ok = false case is not tested. Can you add it ?

Reply:
done